collection/tiffany.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "tiffany",
+    "meaning": "Appearance of God",
+    "aliases": [
+      "tiff"
+    ],
+    "translations": {
+        "gmy": "Θεοφανία"
```

Comment:
Hi @tiffwchen,
Can you please update this translation to use lowercase letters? `θ` instead of `Θ`.